src/routes/staff.js

```diff
@@ -185,4 +185,40 @@ ROUTER.put('/role/:username', (request, response) => {
   sqlRequest.execute('[usp_staff_update_role]', responseHandler);
 });
 
+// -------------------------------------------------------
+// UPDATE EMPLOYEE AVATAR
+// -------------------------------------------------------
+ROUTER.put('/avatar/:employeeId', (request, response) => {
+  let employeeId = request.params.employeeId;
+  let data = request.body;
+  
+  let sqlRequest = new sql.Request();
+
+  sqlRequest.input('id', employeeId);
+  sqlRequest.input('new_avatar', data.new_avatar);
```

Comment:
Please change the name of the property `new_avatar` we should follow camelCase convention for JSON properties